### PR TITLE
R7.28.0.final

### PR DESCRIPTION
**Pull request by mbiarnes**

### CODE REVIEW

File: base/Dockerfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ######################################################################################
-# Dockerfile that provides the image for JBoss Business-Central Workbench 7.27.0.Final
+# Dockerfile that provides the image for JBoss Business-Central Workbench 7.28.0.Final
```
Comment (romartin) [inline base/Dockerfile]: just a minor comment, for future fixing.. IMO we can just remove that comments, are not useful anyway

File: kie-server/base/Dockerfile [hunk 2]
```diff
@@ -1,5 +1,5 @@
 #################################################################################
-# Dockerfile that provides the image for JBoss KIE Server 7.27.0.Final
+# Dockerfile that provides the image for JBoss KIE Server 7.28.0.Final
```
Comment (romartin) [inline kie-server/base/Dockerfile]: same here

File: kie-server/showcase/Dockerfile [hunk 3]
```diff
@@ -1,9 +1,9 @@
 #################################################################################
-# Dockerfile that provides the image for JBoss KIE Server 7.27.0.Final Showcase
+# Dockerfile that provides the image for JBoss KIE Server 7.28.0.Final Showcase
```
Comment (romartin) [inline kie-server/showcase/Dockerfile]: same here

File: server/Dockerfile [hunk 4]
```diff
@@ -1,5 +1,5 @@
 ############################################################################
-# Dockerfile that provides the image for JBoss jBPM Server Full 7.27.0.Final
+# Dockerfile that provides the image for JBoss jBPM Server Full 7.28.0.Final
```
Comment (romartin) [inline server/Dockerfile]: same here

File: base/Dockerfile (hunk 1 above)
Comment (mareknovotny) [reply base/Dockerfile]: we could keep just the name of image there, the version string is probably not used anywhere I guess

**Pull request merged**